enterprise/models.py

```diff
@@ -2037,12 +2037,14 @@ class EnterpriseCustomerReportingConfiguration(TimeStampedModel):
     DATA_TYPE_PROGRESS = 'progress'  # Refers to gathering progress data from Vertica (to be deprecated)
     DATA_TYPE_PROGRESS_V2 = 'progress_v2'  # Refers to gathering progress data from the Analytics Data API
     DATA_TYPE_CATALOG = 'catalog'
+    DATA_TYPE_GRADE = 'grade'
```

Comment:
Can you add DATA_TYPE 'completion' also

Reply:
yes, sure. we have to merge this PR after the edx-enterprise-data.
I will add three of them into a single PR.